.github/workflows/sentry_ruby_test.yml

```diff
@@ -12,20 +12,19 @@ jobs:
     defaults:
       run:
         working-directory: sentry-ruby
-    name: Ruby ${{ matrix.ruby_version }}, options - ${{ toJson(matrix.options) }}
+    name: Ruby ${{ matrix.ruby_version }} & Rack ${{ matrix.rack_version }}, options - ${{ toJson(matrix.options) }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        ruby_version: [2.4, 2.5, 2.6, 2.7, '3.0', jruby]
+        rack_version: [2.0, 3.0]
+        os: [ubuntu-latest]
         include:
-          - { os: ubuntu-latest, ruby_version: 2.4 }
-          - { os: ubuntu-latest, ruby_version: 2.5 }
-          - { os: ubuntu-latest, ruby_version: 2.6 }
-          - { os: ubuntu-latest, ruby_version: 2.7 }
-          - { os: ubuntu-latest, ruby_version: '3.0' }
-          - { os: ubuntu-latest, ruby_version: jruby }
-          - { os: ubuntu-latest, ruby_version: '3.1', options: { rubyopt: "--enable-frozen-string-literal --debug=frozen-string-literal" } }
           - { os: ubuntu-latest, ruby_version: '3.1', options: { without_rack: 1 } }
-          - { os: ubuntu-latest, ruby_version: '3.1', options: { codecov: 1 } }
+          - { os: ubuntu-latest, ruby_version: '3.1', rack_version: 2.0 }
```

Comment:
We can probably get this and the one below by adding `3.1` to the `ruby_version` array?

Reply:
the matrix was doing some weird stuff with includes, so I just left all the 3.1 ones explicitly

Reply:
yea, it just runs these with the simplified matrix, no clue why
![image](https://user-images.githubusercontent.com/6536764/189629713-413b0258-6b49-47ff-bfee-92f5321c0608.png)